webknossos/tests/test_generated_client.py

```diff
@@ -0,0 +1,48 @@
+import calendar
+from datetime import datetime
+
+from webknossos.client import _get_generated_client
+from webknossos.client.generated.api.default import info, list_
+from webknossos.client.generated.models.list_response_200_item import (
+    ListResponse200Item,
+)
+from webknossos.client.generated.types import Unset
+
+
+def time_since_epoch_in_ms() -> int:
+    d = datetime.utcnow()
+    unixtime = calendar.timegm(d.utctimetuple())
+    return unixtime * 1000
+
+
+WK_URL = "https://webknossos.org"
```

Comment:
see my other comment. maybe we don't want to run the tests against webknossos.org? but maybe it's alright.